src/services/create-project.service.js

```diff
@@ -107,18 +107,18 @@ export default (
   });
 };
 
-export const getColorForProject = (projectName: string) => {
-  const possibleProjectColors = [
-    COLORS.hotPink[700],
-    COLORS.pink[700],
-    COLORS.red[700],
-    COLORS.orange[700],
-    COLORS.green[700],
-    COLORS.teal[700],
-    COLORS.violet[700],
-    COLORS.purple[700],
-  ];
+export const possibleProjectColors = [
+  COLORS.hotPink[700],
+  COLORS.pink[700],
+  COLORS.red[700],
+  COLORS.orange[700],
+  COLORS.green[700],
+  COLORS.teal[700],
+  COLORS.violet[700],
+  COLORS.purple[700],
+];
```

Comment:
Yay thanks for thinking to test this!

For someone checking out this file, it might not be clear why this array is being exported. Could you please add a comment like "Exported so that `getColorForProject` can be tested"?

Reply:
Yeah, that's a good idea. I've added the comment now.

Reply:
Thanks!